.prettierignore

```diff
@@ -1,3 +1,2 @@
 node_modules
-src/public/
```

Comment:
Warning: can this include transpiled/minified javascript?

Reply:
It should be ok, as files in `public` are only asset files, nothing generated.